Gemfile

```diff
@@ -30,6 +30,8 @@ gem 'jbuilder', '~> 2.5'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
 
+gem 'rubocop', '~> 0.52.1'
```

Comment:
A good practice is to include a description of the gem for each gem you add. You could use the official description (something from https://rubygems.org/gems/rubocop/versions/0.39.0) or something like `linter`